src/registry/routes/component.js

```diff
@@ -1,11 +1,32 @@
 'use strict';
 
 var GetComponentHelper = require('./helpers/get-component');
+var _ = require('underscore');
 
 module.exports = function(conf, repository){
 
   var getComponent = new GetComponentHelper(conf, repository);
 
+  var setCustomHeaders = function(req, res, componentResponse) {
+    if (req.params.componentVersion === componentResponse.version) {
+      //strong version request
+      res.set(componentResponse.headers);
+    } else {
+      //weak version request, therefore skip the blacklisted headers if any
+      if (_.isEmpty(res.conf.customHeadersToSkipOnWeakVersion || [])) {
```

Comment:
Isn't  `|| []` redundant as you're already doing that in the options' sanitiser?